tools/mo/unit_tests/mo/utils/cli_parser_test.py

```diff
@@ -990,6 +991,118 @@ def test_partial_shapes_freeze_dynamic_negative_case3(self):
+                                  'inp3': np.array(['1.0', '1.0', '2.0', '3.0', '5.0'])}
+        self.assertEqual(list(placeholder_values_res.keys()), list(placeholder_values_ref.keys()))
+        self.assertEqual(inputs_list, ["inp1", "inp2", "inp3"])
+        for i in placeholder_values_ref.keys():
+            assert np.array_equal(placeholder_values_res[i], placeholder_values_ref[i])
+
+    def test_get_shapes_several_inputs_several_partial_shapes6_comma_separator(self):
+        # 0D value for freezing specified using --input command line parameter without shape
+        argv_input = "inp1[3, 1]->[1.0 2.0 3.0],inp2[3.., ..2, 5..10, ?,-1],inp3->False"
+        inputs_list, result, _ = get_placeholder_shapes(argv_input, None)
+        exp_res = {'inp1': PartialShape([3, 1]),
+                   'inp2': PartialShape([(3, np.iinfo(np.int64).max), (0, 2), (5, 10), -1, -1]), 'inp3': None}
+        self.assertEqual(list(exp_res.keys()), list(result.keys()))
+        for i in exp_res.keys():
+            assert np.array_equal(result[i], exp_res[i])
+        placeholder_values_res, input_node_names_res = get_freeze_placeholder_values(argv_input, None)
+        placeholder_values_ref = {'inp1': np.array(['1.0', '2.0', '3.0']), 'inp3': False}
+        self.assertEqual(list(placeholder_values_res.keys()), list(placeholder_values_ref.keys()))
+        self.assertEqual(inputs_list, ["inp1", "inp2", "inp3"])
+        for i in placeholder_values_ref.keys():
+            assert np.array_equal(placeholder_values_res[i], placeholder_values_ref[i])
+
+    def test_get_shapes_several_inputs_several_partial_shapes7_comma_separator(self):
+        # 0D shape and value for freezing specified using --input command line parameter
+        argv_input = "inp1[3,1]->[1.0 2.0 3.0],inp2[3.., ..2,5..10, ?,-1],inp3[]->True"
+        inputs_list, result, _ = get_placeholder_shapes(argv_input, None)
+        exp_res = {'inp1': PartialShape([3, 1]),
+                   'inp2': PartialShape([(3, np.iinfo(np.int64).max), (0, 2), (5, 10), -1, -1]),
+                   'inp3': np.array(False).shape}
+        self.assertEqual(list(exp_res.keys()), list(result.keys()))
+        for i in exp_res.keys():
+            assert np.array_equal(result[i], exp_res[i])
+        placeholder_values_res, input_node_names_res = get_freeze_placeholder_values(argv_input, None)
+        placeholder_values_ref = {'inp1': np.array(['1.0', '2.0', '3.0']), 'inp3': True}
+        self.assertEqual(list(placeholder_values_res.keys()), list(placeholder_values_ref.keys()))
+        self.assertEqual(inputs_list, ["inp1", "inp2", "inp3"])
+        for i in placeholder_values_ref.keys():
+            assert np.array_equal(placeholder_values_res[i], placeholder_values_ref[i])
+
+    def test_get_shapes_and_data_types_partial_shape_with_input_port_comma_separator(self):
+        argv_input = "inp1:1[3,1]->[1.0 2.0 3.0],0:inp2[ 3.. ,..2, 5..10, ?,-1]{i32},inp3:4[5]{f32}->[1.0 1.0 2.0 3.0 5.0]"
```

Comment:
freezing value will contain white-spaces for separation?

Reply:
Yes, freezing value is not of shape type, so I didn't change its format.

Reply:
This approach looks a bit inconsistent. If it does require a big effort, propose to align it as well.

Reply:
Done.